### DISCUSSION TITLE
Installing Infoga

### DISCUSSION THREAD

**Discussion by breezecoolgit** [community/community#23778] [New to GitHub]
<p>I’m having problems installig Infoga. I followed all the steps but when I type python setup.py install, the following message appears:</p>
<p>Traceback (most recent call last):<br>
File “setup.py”, line 8, in &lt;module&gt;<br>
from setuptools import setup<br>
ImportError: No module named setuptools</p>
<p>How can I solve this?</p>
<p>Thanks</p>

**Comment by yamiacat** [comment, marked as answer]
<p>Hi breezecoolgit! It sounds like you may have been looking for support for a specific project? There are a few places you could look for help:</p>
<ul>
<li>its SUPPORT or README file, which may include contact information or an official website</li>
<li>its Issues tab, where you can report bugs</li>
<li>the owner’s GitHub profile, which may include contact information</li>
</ul>
